python/lsst/afw/cameraGeom/utils.py

```diff
@@ -650,11 +665,13 @@ def getCcdInCamBBoxList(ccdList, binSize, pixelSize_o, origin):
         cbbox = ccd.getBBox()
         ex = cbbox.getDimensions().getX()//binSize
         ey = cbbox.getDimensions().getY()//binSize
-        bbox = afwGeom.Box2I(cbbox.getMin(), afwGeom.Extent2I(int(ex), int(ey)))
+        bbox = afwGeom.Box2I(
+            cbbox.getMin(), afwGeom.Extent2I(int(ex), int(ey)))
         bbox = rotateBBoxBy90(bbox, nQuarter, bbox.getDimensions())
         bbox.shift(afwGeom.Extent2I(int(llc.getX()//pixelSize_o.getX()/binSize),
                                     int(llc.getY()//pixelSize_o.getY()/binSize)))
-        bbox.shift(afwGeom.Extent2I(-int(origin.getX()//binSize), -int(origin.getY())//binSize))
+        bbox.shift(afwGeom.Extent2I(-int(origin.getX() //
+                                         binSize), -int(origin.getY())//binSize))
```

Comment:
This one looks inconsistent. I think it should newline after the `binSize),` and use consistent spacing with the `//` in both.